README.md

```diff
@@ -83,6 +83,19 @@ of options:
  * [Libvirt][libvirt]
  * [Cockpit][cockpit]
 
+## Submitting patches
+
+When sending patches to the project, the submitter is required to certify that
+they have the legal right to submit the code. This is achieved by adding a line
+
+    Signed-off-by: Real Name <email@address.com>
```

Comment:
Those links are talking about a different concept - the DCO is not the same as the  "Contributor License Agreement" (CLA) those links are talking about. 

CLAs tend to contain legal language that is either the same as that already contained in the license, or (worse) attempting to grab *more* rights than the license text grants, and then get the user to explicitly sign agreement to the CLA as a way of asserting compliance with the CLA as an indirect way of asserting compliance with the license.  This duplication of legal agreements is redundant, and confusing and (if it tries to grab more rights) it is harmful.

The DCO exists explicitly to avoid the pitfalls of the CLA that those links describe. It does not attempt to repeat/duplicate the legal terms of the license - it just defines a method for directly asserting compliance with the existing license terms at time of each contribution.